trade-finance-manager-ui/server/controllers/case/cancellation/check-details.controller.ts

```diff
@@ -71,13 +77,6 @@ export const postDealCancellationDetails = async (req: PostDealCancellationDetai
       return res.redirect('/not-found');
     }
 
-    const submissionTypeCanBeCancelled = canSubmissionTypeBeCancelled(deal.dealSnapshot.submissionType);
-    const cancellationIncomplete = reason === undefined || !bankRequestDate || !effectiveFrom;
-
-    if (!submissionTypeCanBeCancelled || cancellationIncomplete) {
-      return res.redirect(`/case/${_id}/deal`);
-    }
```

Comment:
Why aren't we validating this now please?

Reply:
I found this was giving some pretty weird behaviour: when the validation wasnt met the browser was redirected to `/case/${_id}/deal` - the same page as when the validation is met & the deal is cancelled.

Given the user can't actually change anythign on this page, it makes the most sense to send the data to the api & if the api rejects it, let the browser render `problem-with-service.njk`.
Given this is a post request, sending weird data requires using Postman, or modifying the HTML, so it should never happen in normal usage